### ISSUE TITLE
Crash-report

### ISSUE THREAD

**Issue by OldKingisdie** [Silentine/GrimoireOfGaia#79]
```
---- Minecraft Crash Report ----

WARNING: coremods are present:
  LoadingPlugin (Quark-r1.4-125.jar)
Contact their authors BEFORE contacting forge

// My bad.

Time: 9/10/18 9:16 PM
Description: Unexpected error

java.lang.NoClassDefFoundError: baubles/api/BaublesApi
	at gaia.items.ItemAccessoryBauble.func_77659_a(ItemAccessoryBauble.java:60)
	at net.minecraft.item.ItemStack.func_77957_a(ItemStack.java:217)
	at net.minecraft.client.multiplayer.PlayerControllerMP.func_187101_a(PlayerControllerMP.java:511)
	at net.minecraft.client.Minecraft.func_147121_ag(Minecraft.java:1595)
	at net.minecraft.client.Minecraft.func_184117_aA(Minecraft.java:2252)
	at net.minecraft.client.Minecraft.func_184118_az(Minecraft.java:2018)
	at net.minecraft.client.Minecraft.func_71407_l(Minecraft.java:1806)
	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:1096)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:397)
	at net.minecraft.client.main.Main.main(SourceFile:123)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:146)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:25)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.jackhuang.hellominecraft.launcher.Launcher.main(Launcher.java:154)
Caused by: java.lang.ClassNotFoundException: baubles.api.BaublesApi
	at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:185)
	at java.lang.ClassLoader.loadClass(Unknown Source)
	at java.lang.ClassLoader.loadClass(Unknown Source)
	... 21 more
Caused by: java.lang.NullPointerException
	at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:176)
	... 23 more


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Client thread
Stacktrace:
	at gaia.items.ItemAccessoryBauble.func_77659_a(ItemAccessoryBauble.java:60)
	at net.minecraft.item.ItemStack.func_77957_a(ItemStack.java:217)
	at net.minecraft.client.multiplayer.PlayerControllerMP.func_187101_a(PlayerControllerMP.java:511)
	at net.minecraft.client.Minecraft.func_147121_ag(Minecraft.java:1595)
	at net.minecraft.client.Minecraft.func_184117_aA(Minecraft.java:2252)
	at net.minecraft.client.Minecraft.func_184118_az(Minecraft.java:2018)

-- Affected level --
Details:
	Level name: MpServer
	All players: 1 total; [EntityPlayerSP['chuxuan'/1120, l='MpServer', x=684.52, y=69.00, z=108.80]]
	Chunk stats: MultiplayerChunkCache: 194, 194
	Level seed: 0
	Level generator: ID 00 - default, ver 1. Features enabled: false
	Level generator options: 
	Level spawn location: World: (224,64,256), Chunk: (at 0,4,0 in 14,16; contains blocks 224,0,256 to 239,255,271), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,0,0 to 511,255,511)
	Level time: 2547 game time, 2547 day time
	Level dimension: 0
	Level storage version: 0x00000 - Unknown?
	Level weather: Rain time: 0 (now: false), thunder time: 0 (now: false)
	Level game mode: Game mode: survival (ID 0). Hardcore: false. Cheats: false
	Forced entities: 116 total; [EntityChicken['鸡'/1280, l='MpServer', x=622.09, y=86.00, z=185.50], EntityBat['蝙蝠'/1281, l='MpServer', x=616.03, y=42.07, z=165.46], EntityItem['item.item.seeds'/1287, l='MpServer', x=683.34, y=62.00, z=61.88], EntitySheep['羊'/1288, l='MpServer', x=646.47, y=67.00, z=86.60], EntitySheep['羊'/1289, l='MpServer', x=643.50, y=67.00, z=83.50], EntitySheep['羊'/1290, l='MpServer', x=643.50, y=67.00, z=85.50], EntitySheep['羊'/1291, l='MpServer', x=652.24, y=67.00, z=87.40], EntityItem['item.item.seeds'/1293, l='MpServer', x=664.77, y=66.00, z=69.35], EntityChicken['鸡'/1298, l='MpServer', x=728.50, y=109.00, z=176.50], EntityWolf['狼'/1299, l='MpServer', x=736.28, y=80.00, z=161.50], EntityWolf['狼'/1300, l='MpServer', x=730.63, y=77.00, z=162.91], EntityWolf['狼'/1301, l='MpServer', x=728.24, y=77.00, z=162.54], EntityWolf['狼'/1302, l='MpServer', x=737.45, y=78.00, z=164.50], EntityChicken['鸡'/1303, l='MpServer', x=728.50, y=77.00, z=171.50], EntityChicken['鸡'/1304, l='MpServer', x=730.50, y=77.00, z=168.50], EntityBat['蝙蝠'/1560, l='MpServer', x=740.23, y=46.00, z=80.32], EntityChicken['鸡'/1305, l='MpServer', x=728.50, y=101.00, z=172.50], EntityBat['蝙蝠'/1563, l='MpServer', x=739.08, y=46.35, z=79.36], EntityBat['蝙蝠'/1564, l='MpServer', x=630.39, y=39.18, z=156.16], EntityChicken['鸡'/1316, l='MpServer', x=750.50, y=153.00, z=185.50], EntityChicken['鸡'/1318, l='MpServer', x=744.13, y=148.00, z=186.46], EntityChicken['鸡'/1319, l='MpServer', x=747.50, y=150.00, z=185.50], EntitySheep['羊'/1323, l='MpServer', x=744.50, y=71.00, z=80.50], EntitySheep['羊'/1324, l='MpServer', x=751.18, y=71.00, z=85.15], EntitySheep['羊'/1325, l='MpServer', x=695.69, y=66.00, z=159.83], EntityGaiaCentaur['半人马'/1326, l='MpServer', x=695.27, y=67.00, z=148.57], EntityZombie['僵尸'/1330, l='MpServer', x=721.13, y=15.00, z=91.30], EntitySpider['蜘蛛'/1331, l='MpServer', x=722.05, y=33.10, z=90.30], EntityHorse['马'/1332, l='MpServer', x=743.05, y=71.00, z=64.14], EntityHorse['马'/1333, l='MpServer', x=746.42, y=71.00, z=71.18], EntityHorse['马'/1334, l='MpServer', x=740.10, y=71.00, z=68.96], EntityHorse['马'/1335, l='MpServer', x=746.86, y=72.00, z=71.90], EntitySheep['羊'/1336, l='MpServer', x=747.50, y=71.00, z=77.50], EntitySheep['羊'/1337, l='MpServer', x=747.50, y=71.00, z=78.50], EntityGaiaCentaur['半人马'/1338, l='MpServer', x=687.90, y=67.00, z=151.68], EntitySkeleton['骷髅'/1339, l='MpServer', x=722.50, y=13.00, z=138.71], EntityGaiaPropFlowerCyan['青色花'/1340, l='MpServer', x=732.50, y=92.00, z=153.50], EntityBat['蝙蝠'/1341, l='MpServer', x=749.75, y=23.10, z=135.91], EntityGaiaPropFlowerCyan['青色花'/1342, l='MpServer', x=745.50, y=89.00, z=140.50], EntityChicken['鸡'/1343, l='MpServer', x=710.50, y=92.00, z=169.50], EntityChicken['鸡'/1344, l='MpServer', x=709.50, y=88.00, z=168.50], EntityChicken['鸡'/1345, l='MpServer', x=708.50, y=86.00, z=168.50], EntityChicken['鸡'/1346, l='MpServer', x=711.50, y=100.00, z=171.50], EntityBat['蝙蝠'/1602, l='MpServer', x=722.45, y=48.64, z=107.66], EntitySheep['羊'/1347, l='MpServer', x=751.18, y=70.00, z=109.16], EntitySheep['羊'/1348, l='MpServer', x=748.27, y=71.00, z=96.50], EntitySheep['羊'/1349, l='MpServer', x=746.47, y=68.00, z=104.55], EntitySheep['羊'/1350, l='MpServer', x=749.50, y=70.00, z=104.50], EntityGaiaSatyress['萨提尔'/1351, l='MpServer', x=747.50, y=69.00, z=107.50], EntityBat['蝙蝠'/1352, l='MpServer', x=739.75, y=26.10, z=132.48], EntityItem['item.item.seeds'/1353, l='MpServer', x=728.93, y=69.00, z=137.86], EntitySheep['羊'/1354, l='MpServer', x=697.55, y=68.00, z=166.55], EntitySheep['羊'/1355, l='MpServer', x=703.48, y=67.00, z=159.82], EntitySheep['羊'/1356, l='MpServer', x=692.76, y=66.00, z=160.50], EntityGaiaSatyress['萨提尔'/1357, l='MpServer', x=697.50, y=71.00, z=170.50], EntityBat['蝙蝠'/1631, l='MpServer', x=674.46, y=54.03, z=152.27], EntityCreeper['爬行者'/1662, l='MpServer', x=614.50, y=33.00, z=29.50], EntityCreeper['爬行者'/1663, l='MpServer', x=620.17, y=33.00, z=33.53], EntitySkeleton['骷髅'/1679, l='MpServer', x=741.50, y=27.00, z=63.50], EntitySkeleton['骷髅'/1680, l='MpServer', x=740.50, y=27.00, z=64.50], EntitySkeleton['骷髅'/1682, l='MpServer', x=742.50, y=30.00, z=88.50], EntityGaiaCreep['苦力普'/1175, l='MpServer', x=676.49, y=54.00, z=100.71], EntityBat['蝙蝠'/1177, l='MpServer', x=609.51, y=27.35, z=171.20], EntitySkeleton['骷髅'/1440, l='MpServer', x=618.50, y=42.00, z=165.50], EntityWitch['女巫'/1457, l='MpServer', x=632.50, y=39.00, z=135.50], EntityCreeper['爬行者'/1209, l='MpServer', x=624.50, y=32.00, z=29.50], EntityPlayerSP['chuxuan'/1120, l='MpServer', x=684.52, y=69.00, z=108.80], EntitySheep['羊'/1224, l='MpServer', x=691.24, y=67.00, z=73.81], EntitySheep['羊'/1225, l='MpServer', x=697.68, y=68.00, z=74.81], EntitySpider['蜘蛛'/1226, l='MpServer', x=718.50, y=30.00, z=60.50], EntityZombie['僵尸'/1228, l='MpServer', x=726.25, y=28.00, z=61.49], EntitySheep['羊'/1229, l='MpServer', x=704.55, y=67.00, z=48.55], EntitySheep['羊'/1230, l='MpServer', x=685.49, y=68.00, z=93.27], EntitySheep['羊'/1231, l='MpServer', x=679.50, y=67.00, z=82.76], EntitySheep['羊'/1232, l='MpServer', x=675.66, y=68.00, z=90.25], EntityGaiaSatyress['萨提尔'/1233, l='MpServer', x=687.86, y=67.00, z=83.94], EntitySpider['蜘蛛'/1234, l='MpServer', x=649.50, y=46.00, z=125.50], EntitySkeleton['骷髅'/1746, l='MpServer', x=652.50, y=50.00, z=151.50], EntitySpider['蜘蛛'/1235, l='MpServer', x=649.50, y=46.00, z=118.50], EntitySkeleton['骷髅'/1747, l='MpServer', x=651.50, y=50.00, z=154.50], EntityItem['item.tile.flower2.oxeyeDaisy'/1236, l='MpServer', x=655.73, y=65.00, z=117.50], EntityItem['item.tile.flower2.oxeyeDaisy'/1237, l='MpServer', x=655.67, y=65.00, z=123.01], EntityItem['item.item.seeds'/1238, l='MpServer', x=655.57, y=65.00, z=122.97], EntityBat['蝙蝠'/1239, l='MpServer', x=605.58, y=28.16, z=173.87], EntitySkeleton['骷髅'/1241, l='MpServer', x=620.50, y=42.00, z=161.50], EntityBat['蝙蝠'/1242, l='MpServer', x=617.75, y=36.82, z=165.45], EntityBat['蝙蝠'/1243, l='MpServer', x=622.52, y=40.10, z=162.25], EntitySheep['羊'/1244, l='MpServer', x=700.50, y=67.00, z=48.50], EntitySheep['羊'/1245, l='MpServer', x=703.09, y=67.00, z=48.61], EntitySheep['羊'/1246, l='MpServer', x=703.55, y=67.00, z=49.55], EntityChicken['鸡'/1247, l='MpServer', x=706.73, y=67.00, z=44.29], EntityChicken['鸡'/1248, l='MpServer', x=706.49, y=68.00, z=53.15], EntityChicken['鸡'/1249, l='MpServer', x=705.50, y=67.00, z=43.50], EntityChicken['鸡'/1250, l='MpServer', x=705.50, y=67.00, z=42.50], EntityZombie['僵尸'/1252, l='MpServer', x=654.54, y=56.00, z=108.73], EntityZombie['僵尸'/1253, l='MpServer', x=652.50, y=58.00, z=102.78], EntitySheep['羊'/1258, l='MpServer', x=714.19, y=68.00, z=92.56], EntitySheep['羊'/1259, l='MpServer', x=713.18, y=69.00, z=81.49], EntitySheep['羊'/1260, l='MpServer', x=673.68, y=69.00, z=115.75], EntitySheep['羊'/1261, l='MpServer', x=683.53, y=67.00, z=143.23], EntitySheep['羊'/1262, l='MpServer', x=675.82, y=64.00, z=132.45], EntitySpider['蜘蛛'/1264, l='MpServer', x=630.70, y=37.10, z=177.30], EntityChicken['鸡'/1265, l='MpServer', x=628.19, y=87.00, z=185.26], EntityChicken['鸡'/1266, l='MpServer', x=628.50, y=87.00, z=187.50], EntityChicken['鸡'/1267, l='MpServer', x=626.40, y=87.00, z=181.79], EntitySheep['羊'/1269, l='MpServer', x=733.44, y=70.00, z=56.80], EntitySheep['羊'/1270, l='MpServer', x=732.50, y=70.00, z=63.50], EntitySheep['羊'/1271, l='MpServer', x=729.50, y=70.00, z=60.50], EntitySheep['羊'/1272, l='MpServer', x=729.50, y=70.00, z=60.50], EntitySheep['羊'/1273, l='MpServer', x=682.13, y=67.00, z=83.50], EntityGaiaSatyress['萨提尔'/1274, l='MpServer', x=697.27, y=68.00, z=87.21], EntityGaiaSatyress['萨提尔'/1275, l='MpServer', x=697.26, y=68.00, z=83.19], EntitySkeleton['骷髅'/1276, l='MpServer', x=666.50, y=45.00, z=125.99], EntityGaiaCreep['苦力普'/1277, l='MpServer', x=658.70, y=53.00, z=117.57], EntityBat['蝙蝠'/1278, l='MpServer', x=654.59, y=24.10, z=149.25], EntitySheep['羊'/1279, l='MpServer', x=672.53, y=64.00, z=129.73]]
	Retry entities: 0 total; []
	Server brand: fml,forge
	Server type: Integrated singleplayer server
Stacktrace:
	at net.minecraft.client.multiplayer.WorldClient.func_72914_a(WorldClient.java:420)
	at net.minecraft.client.Minecraft.func_71396_d(Minecraft.java:2738)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:426)
	at net.minecraft.client.main.Main.main(SourceFile:123)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:146)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:25)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.jackhuang.hellominecraft.launcher.Launcher.main(Launcher.java:154)

-- System Details --
Details:
	Minecraft Version: 1.12.2
	Operating System: Windows 7 (amd64) version 6.1
	Java Version: 1.8.0_162, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 298824528 bytes (284 MB) / 734539776 bytes (700 MB) up to 2134114304 bytes (2035 MB)
	JVM Flags: 7 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -XX:+UseConcMarkSweepGC -XX:+CMSIncrementalMode -XX:-UseAdaptiveSizePolicy -XX:-OmitStackTraceInFastThrow -Xmn128m -Xmx2048m
	IntCache: cache: 0, tcache: 41, allocated: 13, tallocated: 95
	FML: MCP 9.42 Powered by Forge 14.23.4.2705 16 mods loaded, 16 mods active
	States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored

	| State     | ID                 | Version        | Source                                      | Signature                                |
	|:--------- |:------------------ |:-------------- |:------------------------------------------- |:---------------------------------------- |
	| UCHIJAAAA | minecraft          | 1.12.2         | minecraft.jar                               | None                                     |
	| UCHIJAAAA | mcp                | 9.42           | minecraft.jar                               | None                                     |
	| UCHIJAAAA | FML                | 8.0.99.99      | forge-1.12.2-14.23.4.2705.jar               | e3c3d50c7c986df74c645c0ac54639741c90a557 |
	| UCHIJAAAA | forge              | 14.23.4.2705   | forge-1.12.2-14.23.4.2705.jar               | e3c3d50c7c986df74c645c0ac54639741c90a557 |
	| UCHIJAAAA | serverredirect     | 1.3.7          | [转换]ServerRedirect-Forge-MC1.12.2-1.3.7.jar | None                                     |
	| UCHIJAAAA | quark              | r1.4-125       | Quark-r1.4-125.jar                          | None                                     |
	| UCHIJAAAA | autoreglib         | 1.3-18         | AutoRegLib-1.3-18.jar                       | None                                     |
	| UCHIJAAAA | jei                | 4.8.5.138      | jei_1.12.2-4.8.5.138.jar                    | None                                     |
	| UCHIJAAAA | ftblib             | 5.0.2.14       | FTBLib-5.0.2.14.jar                         | None                                     |
	| UCHIJAAAA | ftbutilities       | 5.0.2.14       | FTBUtilities-5.0.2.14.jar                   | None                                     |
	| UCHIJAAAA | grimoireofgaia     | 1.6.0-beta5    | GrimoireOfGaia3-1.12.2-1.6.0-beta5.jar      | None                                     |
	| UCHIJAAAA | ironbackpacks      | 1.12.2-3.0.5-6 | IronBackpacks-1.12.2-3.0.5-6.jar            | None                                     |
	| UCHIJAAAA | minetogether       | unspecified    | minetogether-1.10.2-2.1.3.jar               | None                                     |
	| UCHIJAAAA | minetogetherserver | unspecified    | minetogether-1.10.2-2.1.3.jar               | None                                     |
	| UCHIJAAAA | traverse           | 1.5.5          | Traverse-1.12.2-1.5.5-65.jar                | None                                     |
	| UCHIJAAAA | wts                | 1.1.0          | WTS-1.1.0.jar                               | None                                     |

	Loaded coremods (and transformers): 
LoadingPlugin (Quark-r1.4-125.jar)
  vazkii.quark.base.asm.ClassTransformer
	GL info: ' Vendor: 'Intel' Version: '4.0.0 - Build 10.18.10.4653' Renderer: 'Intel(R) HD Graphics 4000'
	Launched Version: Hello Minecraft! Launcher 2.3.5.7
	LWJGL: 2.9.4
	OpenGL: Intel(R) HD Graphics 4000 GL version 4.0.0 - Build 10.18.10.4653, Intel
	GL Caps: Using GL 1.3 multitexturing.
Using GL 1.3 texture combiners.
Using framebuffer objects because OpenGL 3.0 is supported and separate blending is supported.
Shaders are available because OpenGL 2.1 is supported.
VBOs are available because OpenGL 1.5 is supported.

	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'fml,forge'
	Type: Client (map_client.txt)
	Resource Packs: 
	Current Language: 简体中文 (中国)
	Profiler Position: N/A (disabled)
	CPU: 4x Intel(R) Core(TM) i5-3230M CPU @ 2.60GHz
````

**Comment by Mrbysco** [comment]
Will be fixed next version

**Comment by Silentine** [comment]
Implemented by **Mrbysco** in (1.6.0).
